qcfractal/interface/models/gridoptimization.py

```diff
@@ -53,26 +53,20 @@ def check_step_type(cls, v):
 
         return v
 
-
-class GOOptions(BaseModel):
-    """
-    GridOptimization options
-    """
-    scans: List[ScanDimension]
-    preoptimization: bool = True
-
     class Config:
+        extra = "forbid"
         allow_mutation = False
 
 
-class OptOptions(BaseModel):
+class GOKeywords(BaseModel):
     """
     GridOptimization options
```

Comment:
This docstring now mimics the docstring in OptimizationSpecification, one of them should be changed to better match the true description

Reply:
Not quite `OptimizationSpecification` includes Keywords and other things. The `GOKeywords` are explicitly just keywords.